README.md

```diff
@@ -46,3 +46,29 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+
+## Runtime Analysis, Maxie M. 
+### Time Complexity:
+- **subsets:** there are $2^n$ subsets for $n$ cities
+- **cities in Each Subset**: iterates over $n$ possible cities for each subset
+- **recursive call:** makes recursive call to calculate the cost of remaining cities for each city in a subset
+- **Total number of operations:** $n \times 2^n$
+- **Worst-Case Time Complexity:** $O(n \times 2^n)$
```

Comment:
You're missing something here -- how much work are you doing for each subset and start city?